examples/mapping-example/README.md

```diff
@@ -21,15 +21,17 @@ Next, start up QuickStart and browse to this project folder and login to QuickSt
 
 ## How to test the flows
 
-This project has two flows:
+This project has four flows:
 
 1. The jsonToJson flow ingests and maps a single JSON document to a JSON entity instance
+1. The jsonToXml flow ingests and maps a single JSON document to an XML entity instance
 1. The xmlToJson flow ingests and maps a single XML document to a JSON entity instance
+1. The xmlToXml flow ingests and maps a single XML document to an XML entity instance
 
-The jsonToJson flow demonstrates a wide array of mapping features, while the xmlToJson flow is more focused on a few 
-features specific to XML.
+The jsonToJson and jsonToXml flows demonstrate a wide array of mapping features, while the xmlToJson and xmlToXml flows are more focused on a few
+features specific to XML like attributes and namespaces.
 
-Both flows can be run from the "Flows" page in QuickStart. Each will ingest one document into the staging database and then
+All the flows can be run from the "Flows" page in QuickStart. Each will ingest one document into the staging database and then
```

Comment:
Can you add one note here, which I should have added originally? Where it says "Each of the mapping expressions is then described below. ", add - "Please run each of the flows first to ensure that the mapping step in each flow can be tested."